templates/local_settings.py.j2

```diff
@@ -54,3 +70,13 @@ CATALOGUE = {
 
 MEDIA_ROOT = "/var/www/{{ app_name }}/uploaded"
 STATIC_ROOT = "/var/www/{{ app_name }}/static"
+
+ALLOWED_HOSTS = ['localhost', '{{ server_name }}']
```

Comment:
I agree that things should work through local_settings, but there are deployments where this does not work (perhaps a bug).